### PR TITLE
Update readme with human words

### PR DESCRIPTION
**Pull request by cra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,24 @@
...
+```
+make build
+make run
+```
+
+The command above will autmatically create the required topic (named `test`) and produce data to it. It expects that your broker is accessible as `localhost:9092`
+
+## todo
+
+- [x] Human Readable Readme
```
Comment (cra) [inline README.md]: shall we update the todo list as well?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,24 @@
...
- [] Human Readable Readme
- [] Faster Log Retention
- [] Publish containers
- [] Collect prometheus data
 
+# Lightweight Kafka benchmark tool
+
+The goal of this tool is to efficiently saturate Kafka broker without being the bottleneck itself.
+
+The tool is based ob developed a lightweight tool based on franz-go library and example. This tool is heavily based on [franz-go](https://github.com/twmb/franz-go/tree/master/examples/bench) benchmark example.
```
Comment (epikhinm) [inline README.md]: based **on**

File: README.md [hunk 3]
```diff
@@ -1,6 +1,24 @@
...
+
+You would need [podman](https://podman.io/) installed to build images. See [images/Makefile](images/Makefile) for example configuration
+
+The tool can be run using make
+
+```
+make build
```
Comment (epikhinm) [inline README.md:13]: There is no `make build` and `make run`.
`make cache` for downloading kafka
`make amzn` for building all images of kafka on amzn corretto jvms

File: README.md [hunk 4]
```diff
@@ -1,6 +1,24 @@
...
+make run
+```
+
+The command above will autmatically create the required topic (named `test`) and produce data to it. It expects that your broker is accessible as `localhost:9092`
+
+## todo
```
Comment (epikhinm) [inline README.md]: Let's just delete it. 
Outdated.

**Review by epikhinm — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (cra) [reply README.md:13]: These targets are defined in the top-level Makefile, the targets you are talking about are in the images/Makefile, should we have both of them in the root README?

**Pull request merged**